conformance-suites/2.0.0/conformance2/textures/misc/copy-texture-image.html

```diff
@@ -80,8 +80,12 @@
         "RG8",
         "RGB8",
         "RGBA8",
+        "R8_SNORM",
```

Comment:
Also the added RGB integer formats which are not listed in https://www.khronos.org/opengles/sdk/docs/man3/html/glCopyTexImage2D.xhtml

Assume that list is deliberate, I suggest the testing follow it for now.

I agree with the changes related with float formats that become valid after enabling the extension.